client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -46,143 +46,141 @@ import userFactory from 'lib/user';
 
 const user = userFactory();
 
-const CancelPurchase = localize(
-	React.createClass( {
-		propTypes: {
-			hasLoadedSites: PropTypes.bool.isRequired,
-			hasLoadedUserPurchasesFromServer: PropTypes.bool.isRequired,
-			selectedPurchase: PropTypes.object,
-			selectedSite: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.object ] ),
-		},
-
-		componentWillMount() {
-			if ( ! this.isDataValid() ) {
-				this.redirect( this.props );
-				return;
-			}
-
-			recordPageView( 'cancel_purchase', this.props );
-		},
-
-		componentWillReceiveProps( nextProps ) {
-			if ( this.isDataValid() && ! this.isDataValid( nextProps ) ) {
-				this.redirect( nextProps );
-				return;
-			}
-
-			recordPageView( 'cancel_purchase', this.props, nextProps );
-		},
-
-		isDataValid( props = this.props ) {
-			if ( isDataLoading( props ) ) {
-				return true;
-			}
-
-			const purchase = getPurchase( props ),
-				selectedSite = getSelectedSite( props );
-
-			return selectedSite && purchase && isCancelable( purchase );
-		},
-
-		redirect( props ) {
-			const purchase = getPurchase( props ),
-				selectedSite = getSelectedSite( props );
-			let redirectPath = paths.purchasesRoot();
-
-			if ( selectedSite && purchase && ! isCancelable( purchase ) ) {
-				redirectPath = paths.managePurchase( selectedSite.slug, purchase.id );
-			}
-
-			page.redirect( redirectPath );
-		},
-
-		renderFooterText() {
-			const purchase = getPurchase( this.props ),
-				{ refundText, renewDate } = purchase;
-
-			if ( isRefundable( purchase ) ) {
-				return this.props.translate( '%(refundText)s to be refunded', {
-					args: { refundText },
-					context: 'refundText is of the form "[currency-symbol][amount]" i.e. "$20"',
-				} );
-			}
-
-			const renewalDate = this.props.moment( renewDate ).format( 'LL' );
-
-			if ( isDomainRegistration( purchase ) ) {
-				return this.props.translate( 'Domain will be removed on %(renewalDate)s', {
-					args: { renewalDate },
-				} );
-			}
-
-			return this.props.translate( 'Subscription will be removed on %(renewalDate)s', {
-				args: { renewalDate },
+const CancelPurchase = React.createClass( {
```

Comment:
strange that this component was being double wrapped. do we know why that happened?

Reply:
darn: https://github.com/Automattic/wp-calypso/commit/dc99dd9f55277638184f1a02c0be85bf7cbc7579#diff-65853f422c1d4900968c944e9514bd34R49

The `i18n-mixin-to-localize` codemod didn't recognize that CancelPurchase was already localized